components/director/internal/domain/apptemplate/resolver_test.go

```diff
@@ -671,6 +672,21 @@ func TestResolver_CreateApplicationTemplate(t *testing.T) {
 			appTemplateConv.AssertExpectations(t)
 		})
 	}
+	t.Run("Returns error when application template input is invalid", func(t *testing.T) {
```

Comment:
I would enhance the message to include the invalid part as well:
> Returns error when application template URL input is invalid

Then, the fixture can also include what exactly is invalid there